[2021-02-05 12:33:39,568] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Stage_songs 2021-01-14T09:00:00+00:00 [queued]>
[2021-02-05 12:33:39,574] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Stage_songs 2021-01-14T09:00:00+00:00 [queued]>
[2021-02-05 12:33:39,574] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-02-05 12:33:39,574] {taskinstance.py:881} INFO - Starting attempt 1 of 4
[2021-02-05 12:33:39,574] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-02-05 12:33:39,578] {taskinstance.py:901} INFO - Executing <Task(StageToRedshiftOperator): Stage_songs> on 2021-01-14T09:00:00+00:00
[2021-02-05 12:33:39,582] {standard_task_runner.py:54} INFO - Started process 81429 to run task
[2021-02-05 12:33:39,612] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'udac_example_dag', 'Stage_songs', '2021-01-14T09:00:00+00:00', '--job_id', '1155', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/udac_example_dag.py', '--cfg_path', '/var/folders/ys/hb038j3904j77l4dkb4zk2ym0000gq/T/tmpvogjdghd']
[2021-02-05 12:33:39,613] {standard_task_runner.py:78} INFO - Job 1155: Subtask Stage_songs
[2021-02-05 12:33:39,630] {logging_mixin.py:112} INFO - Running <TaskInstance: udac_example_dag.Stage_songs 2021-01-14T09:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-02-05 12:33:39,688] {stage_redshift.py:62} INFO - Clearing data from destination Redshift table: staging_songs
[2021-02-05 12:33:39,690] {base_hook.py:89} INFO - Using connection to: id: redshift. Host: redshift-cluster-airflow.ca3iipyjqhur.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: None
[2021-02-05 15:40:07,353] {taskinstance.py:1150} ERROR - SSL SYSCALL error: Operation timed out
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/trevorransom/Documents/01 Coding/03 Data Engineering/Udacity/05 Airflow/plugins/operators/stage_redshift.py", line 63, in execute
    redshift.run('DELETE FROM {}'.format(self.table))
  File "/usr/local/lib/python3.9/site-packages/airflow/hooks/dbapi_hook.py", line 164, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.9/site-packages/airflow/hooks/postgres_hook.py", line 93, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/usr/local/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.DatabaseError: SSL SYSCALL error: Operation timed out

[2021-02-05 15:40:07,391] {taskinstance.py:1187} INFO - Marking task as UP_FOR_RETRY. dag_id=udac_example_dag, task_id=Stage_songs, execution_date=20210114T090000, start_date=20210205T103339, end_date=20210205T134007
[2021-02-05 15:40:10,208] {local_task_job.py:102} INFO - Task exited with return code 1
